### PR TITLE
refactor: optimize Fuse.js option

### PR DESCRIPTION
**Pull request by yamash723**
I tried optimizing Fuse.js options to get the results want more.
Try it if you like.

refer: https://fusejs.io/api/options.html

## minMatchCharLength

The default of 1 may include unintended results.

image

## shouldSort

To sort the result list, by score.

## ignoreLocation / distance

Too long strings often contain extraneous characters that can become noise.

⚠️ However, this setting may prevent from getting what used to be able to get.

## keys

Add key for weighted search.
`searchTerm` is for AND search across keys.

### CODE REVIEW

File: src/composables/useSearch.ts [hunk 1]
```diff
@@ -81,7 +81,7 @@ export const useSearch = () => {
       return initialSearchItems.value.slice(0, 100);
     }
 
-    const word = extractOnlySearchWord.value || "h"; // "h" is included in all URLs;
+    const word = extractOnlySearchWord.value || "http"; // "http" is included in all URLs;
```
Comment (yamash723) [inline src/composables/useSearch.ts:84]: neccessary more than 2 charactor because `minMatchCharLength` is set to 2.

**Comment by yamash723** [comment]
> cypress/videos/e2e.ts.mp4

Oh… I committed it by mistake.
I'll delete it later.

File: src/composables/useSearch.ts (hunk 1 above)
Comment (kawamataryo) [reply src/composables/useSearch.ts:84]: Very nice, Thanks!

**Comment by kawamataryo** [comment]
This is a great pull request👍
Thank you!

I'll try out the config of this branch for a week or so, and if all is well, I'll merge it in!

**Comment by kawamataryo** [comment]
I just tried it for a bit and it's so good that I decided to merge it without the need to see how it goes XD.
Thanks so much for the awesome pull request！！！

**Comment by kawamataryo** [comment]
> cypress/videos/e2e.ts.mp4

I'll add this one to gitignore 🙏

**Pull request merged**